### ISSUE TITLE
MonsterLag @ Arena rebuild after a game + while countdown

### ISSUE THREAD

**Issue by Elharter** [Kaos1337/SimpleSpleef-Evolution#33]
I use: R: SimpleSpleef v3.4.2 for CB 1.4.6-R0.1 Dec 22, 2012
CB: 1.4.6

If a game is ended and the arena will be restored there is a big lag for approx. 1-4 seconds.

The same lags appears if we use the shovel while countdown.
Test possible at: mc.villalive.at

thx a lot

**Comment by minecrafter** [comment]
Verify that your configuration is not using another arena restorer.

**Comment by Elharter** [comment]
All installed plugins:

2013-01-14 09:41:01 [INFO] Plugins (23): ModReq, WorldEdit, NoCheatPlus, iConomy
, Inquisitor, mcjobs, Vault, Stargate, PermissionsEx, Modifyworld, Votifier, LWC
, mcMMO, MobArena, WorldGuard, dynmap, Essentials, SimpleRegionMarket, ChatManag
er, AuthMe, Dynmap-Essentials, EssentialsSpawn, SimpleSpleef

**Comment by minecrafter** [comment]
Use pastie.org to paste your configuration file, and stop using PEX.

**Comment by Elharter** [comment]
http://pastie.org/5692347
And i dont stop using Pex, because its one of the accurate plugins ever and it worked perfect with an big compatibility to other plugins.

If i compare PEX with SimpleSpleef Evo.... Pex 757,208 Downloads <> SSE 20,000 Downloads
I think you should be compatible :D

**Comment by minecrafter** [comment]
No, PEX is buggy. [I don't need to link it to you](https://docs.google.com/document/d/1BIgAU7senSG1hY0BjQAQFAnbuOqFhpln7ffN6pkOp4s/edit).

**Comment by Elharter** [comment]
Ok we changed in the last few days our Permission Plugin from PEX to bPermission... like your doc.

Nothing changes.
The Same big monster lag while countdown, and a 2nd big lag if the arena will be restored after a game.

No Resolution.

**Comment by Elharter** [comment]
any news?

**Comment by Elharter** [comment]
what a support
